Sources/DittoHeartbeat/HeartbeatModels.swift

```diff
@@ -0,0 +1,128 @@
+///
+//  HeartbeatModels.swift
+//  
+//
+//  Created by Eric Turner on 2/22/24.
+//
+//  Copyright © 2024 DittoLive Incorporated. All rights reserved.
+
+import Foundation
+
+//MARK: HeartbeatConfig
+public struct DittoHeartbeatConfig {
+    public var id: [String: String]
+    public var secondsInterval: Int
+    public var collectionName: String
+    public var metadata: [String: Any]?
+}
+
+//MARK: HeartbeatInfo
+public struct DittoHeartbeatInfo: Identifiable {
+    public var id: [String: String]
+    public var secondsInterval: Int
```

Comment:
Is there value in making this a `TimeInterval` rather than an `Int`?

Reply:
Walker and I discussed this. Initially it was `TimeInterval`/`Double` but since we expect the use case to be full seconds, and the full interval being on the order of minutes, `Int` seemed more appropriate. What is the value you see for `Double`?

Reply:
Less value for `Double` and more the Apple-y friendliness of `TimeInterval`, but if it can only be whole seconds `Int` or an unsigned int type makes more sense.